examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -257,9 +258,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Please note how `Num Frames==(Num Scenes)*num_frames_to_chop`. \n",
+    "### Storing Predictions\n",
+    "There is a small catch to be aware of when saving the model predictions. In fact, the output of the models are coordinates in the `agent` space and we need to convert them into displacements in the `world` space.\n",
```

Comment:
How about referencing to the corresponding sections in https://github.com/lyft/l5kit/blob/master/coords_systems.md when that PR is merged?

Reply:
I personally would remove every 'the' in front of 'agent' / 'world' space.